app/controllers/v0/sessions_controller.rb

```diff
@@ -88,5 +85,17 @@ def delete_session(token)
       end
     end
     # :nocov:
+
+    def log_errors
+      message = <<-MESSAGE.strip_heredoc
```

Comment:
yes, it looks good to me. All these does is make sure that when you're using the multi-line interpolation, that the leading tabs or spaces are only preserved relative to where to the cursor interpolation block begins or ends, vs the entire line beginning or ending.